src/Symfony/Component/DependencyInjection/Secrets/JweHandler.php

```diff
@@ -0,0 +1,298 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Secrets;
+
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+use Symfony\Component\DependencyInjection\Exception\RuntimeException;
+
+final class JweHandler
+{
+    private $secretsLocation;
+
+    private $publicKey;
+
+    private $privateKeyLocation;
+
+    private $publicKeyLocation;
+
+    private $secrets;
+
+    /**
+     * @param string $fileLocation
+     *
+     * @throws InvalidArgumentException if the fileLocation does not exist in a writable directory
+     */
+    public static function initSecretsFile(string $fileLocation)
+    {
+        if (is_writeable(pathinfo($fileLocation, PATHINFO_DIRNAME))) {
```

Comment:
Use `is_writable($fileLocation))`

Reply:
i was getting false here if it didn't exist